internal/api/middleware/identity.go

```diff
@@ -0,0 +1,26 @@
+package middleware
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+	"github.com/redhatinsights/platform-go-middlewares/identity"
+)
+
+func Identity(next echo.HandlerFunc) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		id := identity.XRHID{}
+		encodedIdentity := c.Request().Header.Get("X-Rh-Identity")
+		decodedIdentity, err := base64.StdEncoding.DecodeString(encodedIdentity)
+		if err != nil {
+			return echo.NewHTTPError(http.StatusUnauthorized, "Unable to decode X-Rh-Identity")
+		}
```

Comment:
After importing module `github.com/redhatinsights/platform-go-middlewares/identity`,  why did you require this method?

Reply:
That middleware is for golang standard libaray net/http and not for the echo framework. I have imported that package just for getting the [identity struct](https://github.com/RedHatInsights/platform-go-middlewares/blob/master/identity/identity.go#L57).